src/modules/navigator/mission_block.cpp

```diff
@@ -749,3 +749,24 @@ MissionBlock::set_idle_item(struct mission_item_s *item)
 	item->autocontinue = true;
 	item->origin = ORIGIN_ONBOARD;
 }
+
+void
+MissionBlock::mission_apply_limitation(mission_item_s &item)
+{
+	/* do nothing if altitude max is negative */
+	if (_navigator->get_land_detected()->alt_max > 0.0f) {
+
+		/* get absolut altitude */
+		float altitude_abs = item.altitude_is_relative
```

Comment:
Instead of the comment `get absolut altitude` just make the variable name `absolute_altitude`.